src/features/experience-matching/ui/company-detail/company-detail.tsx

```
@@ -37,12 +29,32 @@ export const CompanyDetail = ({ nextStep }: { nextStep: () => void }) => {
-            </div>
-          ))}
+          {data &&
+            FIELD_CONFIG.map(({ label, key, format, isLink }) => {
+              const value = data[key];
+              const displayValue = format
+                ? format((value ?? "") as string)
+                : value; // 보여줄 데이터 처리
+              return (
+                <div key={key} className={styles.fieldWrapper}>
+                  <label className={styles.fieldTitle}>{label}</label>
+                  <div className={styles.fieldContent}>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)